packages/lib/src/vis/line/LineVis.module.css

```diff
@@ -4,6 +4,15 @@
   min-width: 0;
   min-height: 0;
   margin: 1rem;
+  --curve-color: var(--h5w-line--color, darkblue);
+  --aux-colors: var(
+    --h5w-line--colorAux,
+    orangered,
+    forestgreen,
+    red,
+    mediumorchid,
+    olive
+  );
```

Comment:
Following the suggestion in https://github.com/silx-kit/h5web/pull/1190#discussion_r962925905